horizon: 50
dt: 0.01
quiet: True
control_axes: 6
full_body: True

noises:
  Q_eta: 1e-12
  Q_nu_r: 1e-12
  Q_f: 1e-12
  R_xy: 1e-10
  R_z: 1e-10
  R_dr: 2e-6
  R_att: 1e-4
  R_linvel: 1e-6
  R_angvel: 0.137e-4
  R_linacc: 0.2029e-4

penalty:
  Q: [
    # Position (x, y, z)
    100.0, 100.0, 100.0,

    # Orientation (roll, pitch, yaw)
    500.0, 500.0, 500.0,

    # Linear Velocity (u, v, w)
    250.0, 250.0, 250.0,

    # Angular Velocity (p, q, r)
    500.0, 500.0, 500.0
  ]

  P: [
    # Position (x, y, z)
    100.0, 100.0, 100.0,

    # Orientation (roll, pitch, yaw)
    500.0, 500.0, 500.0,

    # Linear Velocity (u, v, w)
    250.0, 250.0, 250.0,

    # Angular Velocity (p, q, r)
    500.0, 500.0, 500.0
  ]

  R: [
    # Force (Fx, Fy, Fz)
    0.05, 0.05, 0.05,

    # Torque (Tx, Ty, Tz)
    100.0, 100.0, 100.0
  ]


bounds:
  xmin: [-15., -15., -15., -3.14, -3.14, -3.14, -1.5, -1.5, -1.5, -0.087, -0.087, -0.087]
  xmax: [15., 15., 15., 3.14, 3.14, 3.14, 1.5, 1.5, 1.5, 0.087, 0.087, 0.087]

  # umin: [-0.1, -0.1, -0.1, -0.1, -0.1, -0.1, -0.1, -0.1]
  # umax: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
  # umin: [-250.0, -250.0, -250.0, -250.0, -250.0, -250.0]
  # umax: [250.0, 250.0, 250.0, 250.0, 250.0, 250.0]
  umin: [-20.0, -20.0, -20.0, -20.0, -20.0, -20.0]
  umax: [20.0, 20.0, 20.0, 20.0, 20.0, 20.0]
  # umin: [-35.0, -20.0, -25.0, -1.0, -1.0, -15]
  # umax: [35.0, 20.0, 25.0, 1.0, 1.0, 15]
  # umin: [-35.0, -35.0, -35.0, -10.0, -10.0, -10.0]
  # umax: [35.0, 35.0, 35.0, 10.0, 10.0, 10.0]
  # umin: [-35.0, -35.0, -35.0, -35.0, -35.0, -35.0]
  # umax: [35.0, 35.0, 35.0, 35.0, 35.0, 35.0]
  # umin: [-100.0, -100.0, -100.0, -100.0, -100.0, -100.0]
  # umax: [100.0, 100.0, 100.0, 100.0, 100.0, 100.0]

  # umin: [-5.0, -5.0, -5.0, -5.0, -5.0, -5.0, -5.0, -5.0]
  # umax: [5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0]
  # umin: [-100.0, -100.0, -100.0, -100.0, -100.0, -100.0, -100.0, -100.0]
  # umax: [100.0, 100.0, 100.0, 100.0, 100.0, 100.0, 100.0, 100.0]

  # dumin: [-0.1, -0.1, -0.1, -0.1, -0.1, -0.1]
  # dumax: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
  # dumin: [-1.0, -1.0, -1.0, -1.0, -1.0, -1.0]
  # dumax: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]

  dumin: [-0.5, -0.5, -0.5, -0.5, -0.5, -0.5, -0.5, -0.5]
  dumax: [0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
